### PR TITLE
Pizza

### PR DESCRIPTION
**Pull request by KwikKill**
This PR add a admin pizza handling : 
- Add pizza models
- Add pizza listing and admin endpoints
- Add basic testings for all endpoints
Some changes are draft for further changes :
- Edit Product model to handle pizza transaction
- Create Product on timeslot creation
- Set a user object to pizza order

### CODE REVIEW

File: insalan/pizza/models.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        max_length=2,
+        choices=[
+            ("CA", _("Carte")),
+            ("CH", _("Chèque")),
+            ("ES", _("Espèces")),
+            ("LI", _("Lyfpay")),
+        ],
+        default="CA",
+    )
+    price: models.FloatField = models.FloatField(verbose_name=_("Prix"))
```
Comment (Lymkwi) [inline insalan/pizza/models.py:251]: Prices should be `decimal.Decimal`

File: insalan/pizza/admin.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+
+from .models import Pizza, TimeSlot, Order, PizzaOrder
+
+class PizzaAdmin(admin.ModelAdmin):
+    list_display = ("id", "name", "ingredients")
+    search_fields = ["name", "ingredients"]
```
Comment (Lymkwi) [inline insalan/pizza/admin.py:7]: Maybe being able to search by allergens would be useful too

File: insalan/pizza/models.py [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+        "pizza.Pizza",
+        verbose_name=_("Pizzas commandées"),
+        related_name="pizza_order",
+        through='PizzaOrder',
+    )
+    payment_method: models.CharField = models.CharField(
+        verbose_name=_("Moyen de paiement"),
+        max_length=2,
+        choices=[
+            ("CA", _("Carte")),
```
Comment (Lugrim) [inline insalan/pizza/models.py:244]: Ain't `CB` a more used acronym for "Carte Bancaire"?

File: insalan/pizza/views.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+            serializer.pop("external_product")
+
+            serializers_data.append(serializer)
+
+        return Response(serializers_data)
+
+class OrderList(generics.ListCreateAPIView):
+    """List all orders"""
+    queryset = Order.objects.all()
+    permission_classes = [permissions.IsAdminUser | ReadOnly]
```
Comment (Lymkwi) [inline insalan/pizza/views.py:150]: should we really let anyone read all current orders?

**Review by Lymkwi — COMMENTED** [review]
some little comments here and there

File: insalan/pizza/models.py (hunk 1 above)
Comment (KwikKill) [reply insalan/pizza/models.py:251]: Yep, will be fixed

File: insalan/pizza/admin.py (hunk 2 above)
Comment (KwikKill) [reply insalan/pizza/admin.py:7]: Good idea

File: insalan/pizza/views.py (hunk 4 above)
Comment (KwikKill) [reply insalan/pizza/views.py:150]: well, the serializer used is only giving the ID of the orders but you are right, even this info could be too much.

**Review by Lugrim — COMMENTED** [review]
Once lymkwi's comment are dealt with, it should be ok. Good job, your code quality improved a lot!

**Pull request merged**